<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DeveloperDashboardWindow</class>
 <widget class="QMainWindow" name="DeveloperDashboardWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>623</width>
    <height>602</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SuperGames: Панель управления</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="titleLabel">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Панель разработчика &quot;Разработчик&quot;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>605</width>
         <height>556</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QHBoxLayout" name="topBlock">
          <item>
           <widget class="QPushButton" name="profileButton">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>35</height>
             </size>
            </property>
            <property name="text">
             <string>Профиль</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="addGameButton">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>35</height>
             </size>
            </property>
            <property name="text">
             <string>Добавить игру</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QGroupBox" name="statsBlock">
          <property name="title">
           <string>Статистика</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLabel" name="sellsTodayLabel">
             <property name="text">
              <string>Покупок за сегодня: &lt;u&gt;0&lt;/u&gt;</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="sellsWeekLabel">
             <property name="text">
              <string>Покупок за последнюю неделю: &lt;u&gt;0&lt;/u&gt;</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="sellsAllLabel">
             <property name="text">
              <string>Покупок за всё время: &lt;u&gt;0&lt;/u&gt;</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="reviewsTodayLabel">
             <property name="text">
              <string>Отзывов за сегодня: &lt;u&gt;0&lt;/u&gt;</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="reviewsWeekLabel">
             <property name="text">
              <string>Отзывов за последнюю неделю: &lt;u&gt;0&lt;/u&gt;</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="reviewsAllLabel">
             <property name="text">
              <string>Отзывов за всё время: &lt;u&gt;0&lt;/u&gt;</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="updateStatsButton">
             <property name="text">
              <string>Обновить статистику</string>
             </property>
             <property name="icon">
              <iconset theme="sync-synchronizing">
               <normaloff>.</normaloff>.</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>15</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QGroupBox" name="gamesBlock">
          <property name="title">
           <string>Игры</string>
          </property>
          <layout class="QVBoxLayout" name="gamesVerticalLayout">
           <item>
            <widget class="QLabel" name="noGamesLabel">
             <property name="font">
              <font>
               <pointsize>10</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Нет ни одной выпущенной игры :(
Нажмите на кнопку &quot;Добавить игру&quot;</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>218</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
